src/overlays/actors/ovl_En_Bx/z_en_bx.c

```diff
@@ -9,7 +9,8 @@ void EnBx_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnBx_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnBx_Draw(Actor* thisx, GlobalContext* globalCtx);
 
```

Comment:
theres a little comment block that completed actors have that you should put here. you can leave the description field blank if you arent 100% sure what it is